package.json

```diff
@@ -129,10 +129,12 @@
     "lint": "bin/run-lint"
   },
   "devDependencies": {
+    "@automattic/dops-components": "automattic/dops-components#try/calypso-integration",
```

Comment:
My understanding is that anything that gets transpiled for the client, or isn't required by inheriting modules (of which there currently are none) should be a dev dependency. Also, force of habit.